src/components/dataset/query/QueryViewTable.jsx

```diff
@@ -91,39 +92,10 @@ export class QueryViewTable extends React.PureComponent {
 
   render() {
     const { queryResults, title, token } = this.props;
-    let options = {};
-
-    const columns = [];
-    if (
-      queryResults.rows !== undefined &&
-      queryResults.schema !== undefined &&
-      queryResults.schema.fields !== undefined
-    ) {
-      queryResults.schema.fields.forEach(colData => {
-        const col = {
-          title: colData.name,
-          field: colData.name,
-        };
-
-        columns.push(col);
-      });
-    }
-
-    if (queryResults.totalRows !== undefined) {
-      const numRows = parseInt(queryResults.totalRows, 10);
-      let pageSize = 0;
-      if (numRows > PAGE_SIZE) {
-        pageSize = PAGE_SIZE;
-      } else {
-        pageSize = numRows;
-      }
 
-      options = {
-        pageSize,
-        pageSizeOptions: [pageSize],
-        showFirstLastPageButtons: false,
-      };
-    }
+    const bigquery = new BigQuery({});
```

Comment:
Let's make the default in the constructor an empty object so that we don't need to pass it in here.

I *think* it's something like: `constructor(options = {}) { ... }`